gobblin-service/src/test/java/org/apache/gobblin/service/modules/core/GitFlowGraphMonitorTest.java

```diff
@@ -96,6 +99,9 @@ public void setUp() throws Exception {
     this.gitForPush.commit().setMessage("First commit").call();
     this.gitForPush.push().setRemote("origin").setRefSpecs(this.masterRefSpec).call();
 
+    URI topologyCatalogUri = this.getClass().getClassLoader().getResource("topologyspec_catalog").toURI();
```

Comment:
Should the resource name be make configurable? Like are there any cases where there might be versioning and a specific version should be used?

Reply:
Oops. I see it is the test.